--- config.def.h	2023-09-20 17:35:26.513580489 -0400
+++ config.h	2023-09-20 21:07:39.706831673 -0400
@@ -2,12 +2,12 @@
 
 /* appearance */
 static const char font[]        = "monospace:size=9";
-static const char* normbgcolor  = "#222222";
-static const char* normfgcolor  = "#cccccc";
-static const char* selbgcolor   = "#555555";
-static const char* selfgcolor   = "#ffffff";
-static const char* urgbgcolor   = "#111111";
-static const char* urgfgcolor   = "#cc0000";
+static const char* normbgcolor  = "#282a36";
+static const char* normfgcolor  = "#f8f8f2";
+static const char* selbgcolor   = "#6272a4";
+static const char* selfgcolor   = "#f8f8f2";
+static const char* urgbgcolor   = "#f1fa8c";
+static const char* urgfgcolor   = "#282a36";
 static const char before[]      = "<";
 static const char after[]       = ">";
 static const char titletrim[]   = "...";
